config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,39 @@
+CarrierWave.configure do |config|
+
+  # Use local storage if in development or test
+  if Rails.env.development? || Rails.env.test?
+    CarrierWave.configure do |config|
+      config.storage = :file
+    end
+  end
+
+  # Use AWS storage if in production
+  if Rails.env.production?
+    CarrierWave.configure do |config|
+      config.storage = :fog
+    end
+  end
+
+  config.fog_credentials = {
+    :provider               => 'AWS',                             # required
+    :aws_access_key_id      => 'AKIAIHDXB2LV5GV3UARA',            # required
+    :aws_secret_access_key  => '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',     # required
+    :region                 => 'us-east-1'                        # optional, defaults to 'us-east-1'
+  }
+  config.fog_directory  = 'gallery-image-submissions'               # required
+  #config.fog_host       = 'https://assets.example.com'           # optional, defaults to nil
+  #config.fog_public     = false                                  # optional, defaults to true
+  config.fog_attributes = {'Cache-Control'=>'max-age=315576000'}  # optional, defaults to {}
```

Comment:
Line is too long. [92/80]<br>Unnecessary spacing detected.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.